plugins/coral-plugin-love/client/index.js

```diff
@@ -1,5 +1,10 @@
 import LoveButton from './LoveButton';
 
+import translations from './translations.json';
+import {loadTranslations} from '/coral-i18n/services/i18n';
+
+Promise.all([loadTranslations(translations)]);
```

Comment:
Yes. I was going to remove that and then I forgot when everything was working fine :/